docs/component-guides/kserve.md

```diff
@@ -0,0 +1,129 @@
+    1. ```
+        kustomize build awsconfigs/common/istio-ingress/overlays/https | kubectl apply -f -
+        ``` 
+1. Get the ALB Address
+    1. ```
+        kubectl get ingress -n istio-system istio-ingress
+        NAME            CLASS    HOSTS   ADDRESS                                                                  PORTS   AGE
+        istio-ingress   <none>   *       xxxxxx-istiosystem-istio-2af2-1100502020.us-west-2.elb.amazonaws.com   80      15d
+        ```
+
+### Add DNS records for the staging subdomain with ALB address
+
+When ALB is ready/updated, copy the ADDRESS of that load balancer and create a CNAME entry to it in Route53 under subdomain (`platform.example.com`) for `*.staging.platform.example.com`
+
+### Run a sample InferenceService
+
+1. [Known Issue] Namespaces created by Kubeflow profile controller have a missing authorization policy which prevent the KFServing predictor and transformer to work([kserve/kserve#1558](https://github.com/kserve/kserve/issues/1558), [kubeflow/kubeflow#5965](https://github.com/kubeflow/kubeflow/issues/5965)). You will need to create the `AuthorizationPolicy` as mentioned in [#82 (comment)](https://github.com/awslabs/kubeflow-manifests/issues/82#issuecomment-1068641378) as a workaround until this is resolved. Verify the policies have been created by listing the `authorizationpolicies` in the `istio-system` namespace:
+    1. ```
+        kubectl get authorizationpolicies -n istio-system
+        ```
+1. Create a sample sklearn InferenceService using a [sample](https://github.com/kubeflow/kfserving-lts/blob/release-0.6/docs/samples/v1beta1/sklearn/v1/sklearn.yaml) from the KFserving repository and wait for it to be `READY`
+    1. ```
+        kubectl apply -n staging -f https://raw.githubusercontent.com/kubeflow/kfserving-lts/release-0.6/docs/samples/v1beta1/sklearn/v1/sklearn.yaml
+        ```
+1. Check `InferenceService` status. Once it is `READY`, get the URL and use it for sending prediction request
+    ```
+    kubectl get inferenceservices sklearn-iris -n staging
+
+    NAME           URL                                                READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                    AGE
+    sklearn-iris   http://sklearn-iris.staging.platform.example.com   True           100                              sklearn-iris-predictor-default-00001   3m31s
+    ```
+1. Send an inference request using the sample python script below based on your auth provider:
+    - **[Cognito]** Run the [sample inference Python script](../../tests/e2e/utils/kserve/inference_sample_cognito.py) in by substituting the values for `KUBEFLOW_DOMAIN`(e.g. platform.example.com), `PROFILE_NAMESPACE`(e.g. staging), custom HTTP header name(e.g. `x-api-key`) and value(e.g. `token1`) according to the values configured in [ingress section](#create-ingress)
+        1. Output would look like:
+            ```
+            python tests/e2e/utils/kserve/inference_sample_cognito.py
```

Comment:
I know this is a small script for quick testing purposes, but would it make sense to pull `KUBEFLOW_DOMAIN` and `PROFIILE_NAMESPACE` from env to keep with the flow of the README without having to modify the python file to test?

Reply:
done